docs/LogGuide.md

```diff
@@ -1706,14 +1747,36 @@ at the moment the action is "locked in".
 > so I would need to do a lot of work-arounds."
 
 Structure:
-`25:[Player ObjectId]:[Sequence Number]:[Current HP]:[Max HP]:[Current MP]:[Max MP]:[Current TP]:[Max TP]:[Position X]:[Position Y]:[Position Z]:[Facing]:[packet data thereafter]`
-
+`25:[Player ObjectId]:[Sequence Number]:[Current HP]:[Max HP]:[Current MP]:[Max MP]:[Unused]:[Unused]:[Position X]:[Position Y]:[Position Z]:[Facing]:[packet data thereafter]`
+`37|[timestamp]|[targetId]|[targetName]|[Sequence Number]|[target hp]|[target max hp]|[target mp]|[target max mp]|[unused]|[unused]|[x]|[y]|[z]|[heading]|[unknown]|[unknown]|[unknown]|[unknown]|[unknown]|[unknown]|[unknown]|[unknown]|16985ce13b1e6fa6`
 Examples:
 
 ```log
 25:12345678:PlayerOne:0000132A:33635:35817:10000:10000:0::0.3841706:-207.8767:2.901163:-3.00212:03E8:2500:0:01:03000000:0:0:E0000000:
+37|2022-02-12T20:06:22.0750000-08:00|1234ABCD|Foo Bar|0004ADA6|473|473|73|10000|0||38.99|101.76|-8.00|1.11|0800|0|04DE|01|020008B2|BFD20004|0|107284AA|16985ce13b1e6fa6
 ```
 
+#### Tracking Ability Resolution
+
+Unfortunately, this is not particularly clean. For one, while the server does tell the game when an action is resolved,
+it does not tell the game when an action will not resolve (ghosting). However, the caster dying or target becoming
+untargetable is usually a decent indicator that something will not resolve.
+
+Note that AoE abilities may have the same sequence ID for all targets hit. Thus, you need to key off of both the
+sequence ID, *and* the target.
+
+In addition, you don't always get a 37-line. For example, a heal that is pure overheal will not generate one.
+
+#### HP Values
+
+Sometimes, only the current HP is present, rather than current and max. In this case, it should be assumed that the max
+HP is unchanged.
+
+The current HP value is actually an update to the target's HP. In other words, it is the target's HP after this ability
+resolves, and thus is the most up to date HP value for the target. Line 38 and 39 also work this way, while other lines
+pull HP data from memory, and do not necessarily indicate HP changes in the first place. Thus, if you only care about
+HP, then you should ideally use only 37/38/39 lines.
```

Comment:
EffectResult and StatusList always contain both the current and max resources. UpdateHPMP (line type 39) contains both as well but only the current ones are correct, the rest come from ACT memory reading.

Reply:
Good to know, thanks. *Probably* not the biggest issue since max HP/MP updates are less common than current.